cmd/snap/cmd_ack.go

```diff
@@ -50,7 +50,9 @@ func init() {
 	addCommand("ack", shortAckHelp, longAckHelp, func() flags.Commander {
 		return &cmdAck{}
 	}, nil, []argDesc{{
+		// TRANSLATORS: This needs to be wrapped in <>s.
 		name: i18n.G("<assertion file>"),
+		// TRANSLATORS: This needs to start with an uppercase letter (unicode OK).
```

Comment:
I don't know if we can/should mandate this. I have no idea if there are languages in which this may be incorrect to start with an uppercase. Maybe we should just drop the check or somehow move it into tests?

Reply:
how about:
* s/needs/should, if appropriate,/
* change the check from `! IsUpper` to `IsLower` (because both are false for e.g. the first rune in `网站上的用户的电子邮件`), and explicitly check it isn't `login.ubuntu.com` (maybe even check if it isn't urlish, to not have to tweak the check if we add urls to our descs)